app_dart/lib/src/foundation/utils.dart

```diff
@@ -85,10 +85,33 @@ Future<RepositorySlug> repoNameForBuilder(List<Map<String, dynamic>> builders, S
 }
 
 /// Gets supported luci builders based on [bucket] via GitHub http request.
+///
+// TODO(keyonghan): to be removed when APIs are updated to call getRepoBuilders, https://github.com/flutter/flutter/issues/62429
 Future<List<Map<String, dynamic>>> getBuilders(HttpClientProvider branchHttpClientProvider, Logging log,
     GitHubBackoffCalculator gitHubBackoffCalculator, String bucket) async {
   final String filename = bucket == 'try' ? 'luci_try_builders.json' : 'luci_prod_builders.json';
-  String builderContent = await remoteFileContent(branchHttpClientProvider, log, gitHubBackoffCalculator, filename);
+  String builderContent = await remoteFileContent(
+      branchHttpClientProvider, log, gitHubBackoffCalculator, '/flutter/cocoon/master/app_dart/dev/$filename');
+  builderContent ??= '{"builders":[]}';
+  Map<String, dynamic> builderMap;
+  try {
+    builderMap = json.decode(builderContent) as Map<String, dynamic>;
+  } on FormatException catch (e) {
+    log.error('error: $e');
```

Comment:
When we see this in server logs, it's hard to tell the problem without reading source code. Please make error message more informative.

Reply:
This is not needed since we have implemented json validation in pre-submit tasks. Removed.